media-libs/oyranos/oyranos-9999.ebuild

```diff
@@ -80,18 +88,37 @@ multilib_src_configure() {
 	local libdir=$(get_libdir)
 	local mycmakeargs=(
 		-DLIB_SUFFIX=${libdir#lib}
+		-DUSE_SYSTEM_ELEKTRA=YES
+		-DUSE_SYSTEM_YAJL=YES
+		$(cmake-utils_use_find_package cairo)
+		$(cmake-utils_use_find_package cups)
+		$(cmake-utils_use_find_package doc Doxygen)
+		$(cmake-utils_use_find_package exif Exif2)
+		$(cmake-utils_use_find_package jpeg)
+		$(cmake-utils_use_find_package raw LibRaw)
+		$(cmake-utils_use_find_package tiff)
+		$(cmake-utils_use_find_package X X11)
+	)
 
-		$(usex cairo -DWANT_CAIRO=1 "")
-		$(usex cups -DWANT_CUPS=1 "")
-		$(usex doc -DWANT_HTML=1 "")
-		$(usex exif -DWANT_EXIV2=1 "")
-		$(usex raw -DWANT_LIBRAW=1 "")
-		$(usex X -DWANT_X11=1 "")
+	# prefer Qt5
+	if ! use qt5 ; then
+		use qt4 && mycmakeargs+=( $(cmake-utils_use_use qt4) )
```

Comment:
Hmm, so USE_QT4 only forces Qt4 over Qt5? Does it care if Qt4 is found at all? Maybe, in the common block:

    -DUSE_QT4=$(usex '!qt5')

This would be simpler if it doesn't cause failures when both Qt4 & Qt5 are disabled.